owtf/api/handlers/api_token.py

```diff
@@ -0,0 +1,42 @@
+"""
+owtf.api.handlers.auth
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+"""
+from owtf.api.handlers.base import APIRequestHandler
+from owtf.api.handlers.jwtauth import jwtauth
+from uuid import uuid4
+from owtf.models.api_token import ApiToken
+import jwt
+secret_key = "my_secret_key"
```

Comment:
can this be defined in settings and imported here? Its a bad practice to hard-code a secret in code.

Reply:
![image](https://user-images.githubusercontent.com/37950602/121947036-38095180-cd73-11eb-8ad3-758c62f06ffd.png)
Added it in settings.py itself and imported it in every file wherever needed.